### PR TITLE
Eclipse removal

### PR DESCRIPTION
**Pull request by KirbyKatcher**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: pipelines/pipelines/cli/build-branch.yaml [hunk 1]
```diff
@@ -467,16 +467,18 @@ spec:
...
-      - branch-eclipse
+      - branch-isolated
       - -n
       - galasa-build
       - --prefix-name 
-      - trigger-eclipse-$(params.toBranch)
+      - trigger-isolated-$(params.toBranch)
       - --param
-      - fromObrBranch=$(params.toBranch)
+      - fromObrBranch=$(params.fromObrBranch)
```
Comment (jadecarino) [inline pipelines/pipelines/cli/build-branch.yaml:476]: This needs to be `fromObrBranch=$(params.toBranch)`

**Review by jadecarino — CHANGES_REQUESTED** [review]
One small change otherwise looks good!

**Review by jadecarino — APPROVED** [review]

**Pull request merged**